.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: AI Code Reviewer
+
+on: [pull_request]
+
+    # types:
+    #   - opened
+    #   - synchronize
+# permissions: write-all
+jobs:
+  build:
```

Comment:
The job name has been changed from `review` to `build`. Make sure this name accurately represents the job's purpose.